src/cookbooks/additional_props.md

```diff
@@ -0,0 +1,46 @@
+# Additional Properties
+
+If some field is present in a valid received [ping](../concepts/terminology.md#ping),
+but is not present in the ping's [schema](../concepts/terminology.md#schema),
+it doesn't have its own column to be placed into during
+[ingestion](../concepts/terminology.md#ingestion).
+Instead, those fields remain as raw JSON and are placed in the
+`additional_properties` column of the ping's table or view.
+
+This can happen for a variety of usually-temporary problems like:
+
+- The latest schema hasn't yet been deployed
+- There is a bug in sending or receiving data
```

Comment:
```suggestion
- [mozilla-pipeline-schemas](https://github.com/mozilla-services/mozilla-pipeline-schemas/commit/0a24af0b56b6645906858ed02b20f977a4fb3842) was not updated after a change in the data which was submitted (this shouldn't happen with [Glean](../concepts/glean.md), but can happen with some legacy data, for example the legacy desktop telemetry envionment).
```

Reply:
Why this commit in particular? (Also, I'd specify "legacy Firefox Desktop telemetry environment")

Reply:
Eugh, I did not mean to link a specific commit there. Sorry, long day :)

Reply:
Adjusted the suggestion